src/dbnode/persist/fs/mmap_util.go

```diff
@@ -85,8 +85,8 @@ func validateAndMmapFile(
 
 	if calculatedDigest := digest.Checksum(mmapDescriptor.Bytes); calculatedDigest != expectedDigest {
 		mmap.Munmap(mmapDescriptor)
-		return mmap.Descriptor{}, fmt.Errorf("expected summaries file digest was: %d, but got: %d",
-			expectedDigest, calculatedDigest)
+		return mmap.Descriptor{}, fmt.Errorf("expected file %s digest was: %d, but got: %d",
```

Comment:
nit: should this be `%s file` rather than `file %s`?

Reply:
👍